docs/development/ide-setup.adoc

```diff
@@ -0,0 +1,35 @@
+= Setting up your IDE for ALMighty Development
+
+== Visual Studio Code [[vscode]]
+
+=== Installation [[vscodeinstallation]]
+
+Download and install the appropriate distribution of Visual Studio Code from link:https://code.visualstudio.com/download[Visual Studio Code Download site] 
+
+=== Configuration [[vscodeconfiguration]] 
+
+* Launch Visual Studio Code.
+* Go to the "File" menu and click on  "Open Folder"
+* Select the directory `almightly-core` inside your cloned local repository directory path.
+* Configure your $GOPATH for the IDE by placing the file `.vscode/settings.json` inside `almighty-core`. Set your `go.gopath` according to your environment. The `go.goroot` should be the path of your Golang installation directory.
+```
+{
+    "go.buildOnSave": false,
+    "go.lintOnSave": true,
+    "go.vetOnSave": true,
+    "go.buildTags": "",
+    "go.buildFlags": [],
+    "go.lintFlags": [],
+    "go.vetFlags": [],
+    "go.coverOnSave": false,
+    "go.useCodeSnippetsOnFunctionSuggest": false,
+    "go.formatOnSave": true,
+    "go.formatTool": "goreturns",
+    "go.goroot": "/usr/lib/golang",
+    "go.gopath": "/path/to/checkedout/repo"
```

Comment:
I think you should add a comment here to explain the details and leave the path with `/replace/me`.

The setup is for Linux only btw. so you should add another comment on the `go.goroot` line or have one comment that deals with both.